src/features/projectsV2/ProjectsMap/microComponents/FireLocations.tsx

```diff
@@ -0,0 +1,74 @@
+import type { ReactElement } from 'react';
+import type {
+  FireFeature,
+  FireFeatureCollection,
+} from '../../../common/types/fireLocation';
+
+import { APIError } from '@planet-sdk/common';
+import { useRouter } from 'next/router';
+import { useEffect, useState } from 'react';
+import { Marker, Source } from 'react-map-gl-v7/maplibre';
+import { getRequest } from '../../../../utils/apiRequests/api';
+import FirePopup from '../FirePopup';
+
+export default function FireLocations(): ReactElement {
+  const { query } = useRouter();
+
+  const { site } = query;
+
+  const [fireFeatures, setFireFeatures] = useState<FireFeature[]>([]);
+
+  useEffect(() => {
+    if (!site) return;
+
+    const fetchFires = async () => {
+      try {
+        const qs = new URLSearchParams();
+        qs.append('remoteId', site as string);
+        qs.append('span', '30d');
```

Comment:
@rupamkairi 
1. could you expand `qs` to `queryParams` or `searchParams`? Makes the code more readable without being too long either.
2. `30d` - could you store this in a variable with an appropriate name (e.g. const ALERT_DURATION = `30 d`) - to avoid having magic strings in the code.